### PR TITLE
Editorial: define attribute list in terms of Infra's list

### PR DESCRIPTION
**Pull request by annevk**
***
[Preview](https://dom.spec.whatwg.org/branch-snapshots/annevk/attribute-list-infra/) | [Diff](https://s3.amazonaws.com/pr-preview/whatwg/dom/3078a9d...53f20eb.html)

### CODE REVIEW

**Comment by annevk** [comment]
See also https://github.com/whatwg/infra/issues/103. Could wait for that potentially, but not sure.

File: dom.bs [hunk 1]
```diff
@@ -1270,7 +1270,7 @@ for discussion).
 <ol>
  <li><p>If <var>event</var>'s <a>stop propagation flag</a> is set, then return.
 
- <li><p>Let <var>listeners</var> be a new <a>list</a>.
+ <li><p>Let <var>listeners</var> be a new <a for=/>list</a>.
```
Comment (domenic) [inline dom.bs]: Can't you use link defaults so you don't have to do this?

Comment (annevk) [inline dom.bs]: I don't like link defaults. What was that other option we thought of enabling at some point for WHATWG?

Comment (domenic) [inline dom.bs]: Explicit for, which requires for="" always if the source has it.

I think link defaults is good and honestly we could enable link defaults to infra for all of WHATWG on things like list/string.

Anchors is the bad one.

Comment (annevk) [inline dom.bs]: You mean we could put link defaults at a higher level than the source for WHATWG in Bikeshed somehow? That might be nice.

Comment (annevk) [inline dom.bs]: (The problem with link defaults is that what we consider a shared ecosystem is in fact using contradictory terms after all. So ideally one of the sides in the dispute changes somehow.)

**Comment by annevk** [comment]
Used link-defaults, but guess I'll wait to see if "replace" becomes a thing.

**Pull request merged**